src/main/java/core/basesyntax/service/impl/ReportServiceImpl.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.ReportService;
+import java.util.Map;
+
+public class ReportServiceImpl implements ReportService {
+    private static final String COMMA_SEPARATOR = ",";
+    private static final String LINE_SEPARATOR = System.lineSeparator();
+
+    @Override
+    public String createReport(Map<String, Integer> data) {
+        StringBuilder stringBuilder = new StringBuilder();
+        stringBuilder.append("fruit")
```

Comment:
You can create constant with name HEADER, wich will contain header string and append this constant to stringBuilder, it will be more clear in my mind  because now "fruit" and "quantity" kind of magic strings